website/common/locales/en/settings.json

```diff
@@ -40,10 +40,12 @@
     "xml": "(XML)",
     "json": "(JSON)",
     "customDayStart": "Custom Day Start",
+    "adjustment": "Adjustment",
+    "dayStartAdjustment": "Day Start Adjustment",
     "sureChangeCustomDayStartTime": "Are you sure you want to change your Custom Day Start time? Your Dailies will next reset the first time you use Habitica after <%= time %>. Make sure you have completed your Dailies before then!",
     "customDayStartHasChanged": "Your custom day start has changed.",
     "nextCron": "Your Dailies will next reset the first time you use Habitica after <%= time %>. Make sure you have completed your Dailies before this time!",
-    "customDayStartInfo1": "Habitica defaults to check and reset your Dailies at midnight in your own time zone each day. You can customize that time here.",
+    "customDayStartInfo1": "Habitica defaults to check and reset your Dailies at 12:00 AM in your own time zone each day. Shift when your day switches over past the default time.",
```

Comment:
@Tressley

Reply:
"12:00 AM" isn't the best term to use because it means different things to different people. https://en.wikipedia.org/wiki/12-hour_clock#Confusion_at_noon_and_midnight shows some examples.

Can we keep using "midnight" for clarity?

Reply:
By the way, changing the words does mean that all the translations have to be changed, and also all the wikis (we typically include screenshots and/or text copies of words on the site). That's not worth considering if the words really do need to be changed, but if it's decided that a change isn't needed, that means less work for the contributors.

Reply:
We were experimenting with new language and I agree that the new copy is reading a little odd. Additionally, we were using "12:00 AM" to connect from the time actually selected from the list. Taking feedback into account, here is revised copy that flows better and reverts back to using midnight:
> Habitica checks and resets your Dailies at midnight in your own time zone each day. You can adjust when that happens past the default time here.

Reply:
some context for the changes: we wanted to edit the wording and UI of the feature to make it more clear that its pushing the time back from the original start time, due to user reports of people setting their day start to later in the day and being confused why their dailies aren't showing on their 'proper' days. so we do think asking for another pass of translation is worth it to hopefully clear up some confusion.

Reply:
Could it say "12:00 midnight in your own time zone"?

Reply:
I think we'll move forward with just "midnight" for now and adjust should we hear about any static.